File: client/bcephal-client-model/Reconciliation/AutoRecoEditorData.cs
using Bcephal.Models.Base;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Reconciliation
{
    public class AutoRecoEditorData : EditorData<AutoReco>
    {

        public ObservableCollection<Nameable> ReconciliationModels { get; set; }

        public ObservableCollection<Nameable> Routines { get; set; }

    }
}

File: server/bcephal-reco/src/main/java/com/moriset/bcephal/reconciliation/domain/AutoRecoEditorData.java
/**
 * 
 */
package com.moriset.bcephal.reconciliation.domain;

import java.util.ArrayList;
import java.util.List;

import com.moriset.bcephal.domain.EditorData;
import com.moriset.bcephal.domain.Nameable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author Joseph Wambo
 *
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper=true)
public class AutoRecoEditorData extends EditorData<AutoReco> {

	private List<Nameable> reconciliationModels;
	
	private List<Nameable> reconciliationJoinModels;
	
	private List<Nameable> routines = new ArrayList<Nameable>();
	
}
